### PR TITLE
SPEC: update package version to 1.0-8

### PR DESCRIPTION
**Pull request by bercarug**
Included the latest changes in the SPEC file changelog
and updated the package version to 1.0-8.

Signed-off-by: Gabriel Bercaru <bercarug@amazon.com>

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: SPECS/aws-nitro-enclaves-cli.spec [hunk 1]
```diff
@@ -159,6 +159,13 @@ rm -rf %{ne_log_dir}
 %{ne_include_dir}/*
 
 %changelog
+* Tue Nov 24 2020 Gabriel Bercaru <bercarug@amazon.com> - 1.0.8
```
Comment (andraprs) [inline SPECS/aws-nitro-enclaves-cli.spec:162]: s/1.0.8/1.0-8/

File: SPECS/aws-nitro-enclaves-cli.spec [hunk 2]
```diff
@@ -159,6 +159,13 @@ rm -rf %{ne_log_dir}
...
+- Added third_party directory with linuxkit credit
+- Improved 'insufficient resources' error messages
+- Updated the allocator service
+- Enfore an enclave memory lower limit of 4x the size of the EIF file
```
Comment (andraprs) [inline SPECS/aws-nitro-enclaves-cli.spec:166]: s/Enfore/Enforce.

**Review by andraprs — APPROVED** [review]

File: SPECS/aws-nitro-enclaves-cli.spec (hunk 1 above)
Comment (bercarug) [reply SPECS/aws-nitro-enclaves-cli.spec]: Done in the latest revision.

File: SPECS/aws-nitro-enclaves-cli.spec (hunk 2 above)
Comment (bercarug) [reply SPECS/aws-nitro-enclaves-cli.spec]: Done in the latest revision.

**Review by andraprs — APPROVED** [review]

**Comment by petreeftime** [comment]
Cargo.toml still lists the version as 0.1.0, plese keep them in sync.

**Review by andraprs — DISMISSED** [review]

**Review by alexggh — DISMISSED** [review]

**Review by andraprs — APPROVED** [review]

**Review by petreeftime — APPROVED** [review]

**Review by andraprs — APPROVED** [review]

**Review by neacsu — APPROVED** [review]

**Pull request merged**